packages/theme/src/Components/SearchableList/SearchableList.tsx

```diff
@@ -0,0 +1,157 @@
+ *           searchKey={['name']}
+ *           itemRender={ListItem}
+ *      />
+ * )
+ */
+export const SearchableList = <T,>({
+    key,
+    data,
+    placeholder,
+    onSelect,
+    onSearch,
+    searchKey,
+    itemRender,
+    FixedSizeListProps,
+}: MaskSearchableListProps<T>) => {
+    const [keyword, setKeyword] = useState('')
+    const classes = useStyles()
+
+    //#region fuse
+    const fuse = useMemo(
+        () =>
+            new Fuse(data, {
+                shouldSort: true,
+                threshold: 0.45,
+                minMatchCharLength: 1,
+                keys: searchKey ?? Object.keys(data.length > 0 ? data[0] : []),
+            }),
+        [data],
+    )
+    //#endregion
+
+    //#region create searched data
+    const readyToRenderData = useMemo(() => {
+        if (!keyword) return data
+        const filtered = [...(onSearch ? onSearch(data, keyword) : []), ...fuse.search(keyword).map((x: any) => x.item)]
+        return key ? uniqBy(filtered, (x) => x[key]) : filtered
+    }, [keyword, fuse, data])
+    //#endregion
+
+    return (
+        <div className={classes.container}>
+            <TextField
+                className={classes.textField}
+                placeholder={'Search'}
+                autoFocus
+                fullWidth
+                InputProps={{
+                    className: classes.searchInput,
+                    startAdornment: (
+                        <InputAdornment position="start">
+                            <Search />
+                        </InputAdornment>
+                    ),
+                }}
+                onChange={(e) => setKeyword(e.currentTarget.value)}
+            />
+            {placeholder && (
+                <Typography className={classes.placeholder} color="textSecondary">
+                    {placeholder}
+                </Typography>
+            )}
+            {!placeholder && (
+                <div className={classes.list}>
+                    <FixedSizeList
+                        width="100%"
+                        height={300}
+                        overscanCount={5}
+                        itemSize={60}
+                        itemData={{
+                            dataSet: readyToRenderData,
+                            onSelect: onSelect,
+                        }}
+                        itemCount={readyToRenderData.length}
+                        {...FixedSizeListProps}>
+                        {(props) => <MaskSearchableItemInList<T> {...props}>{itemRender}</MaskSearchableItemInList>}
+                    </FixedSizeList>
+                </div>
+            )}
+        </div>
+    )
+}
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+        paddingBottom: theme.spacing(4),
+    },
+    textField: {
+        '&>div': {
+            borderRadius: theme.spacing(1),
+            background: MaskColorVar.normalBackground,
+        },
+    },
+    searchInput: {
+        padding: '-6px',
```

Comment:
`padding` doesn't support negative values, do you mean `6px`?

Reply:
This is not valid, I will remove it.